index.js

```diff
@@ -13,12 +14,13 @@ const { errorHandler } = require("./middleware/errorHandler.mw");
 const logPath = path.join(__dirname, "logs", "http.log");
 const port = process.env.PORT || 3000;
 const { experimentRouter } = require("./router/experiment.router");
-const { testRouter } = require("./router/external.routes");
+// const { testRouter } = require("./router/external.routes");
 const { userRouter } = require("./router/user.routes");
 const statsRouter = require("./router/stats.router");
 const {goalRouter} = require("./router/goal.router");
-const { experimentStatusUpdate } = require("./Service/cron.job");
+const {experimentStatusUpdate} = require("./middleware/cron.job");
 
+experimentStatusUpdate();
```

Comment:
also the file index.js is changed to app.js The code should be updated accordingly